maquetado/index.html

```diff
@@ -8,6 +8,65 @@
     <link rel="stylesheet" type="text/css" href="../application.css">
 </head>
 <body>
-    <h1 class="title">Front-End Tools</h1>
+	<h1 class="title">Hello World</h1>
```

Comment:
Here you should have the title with the image behind that you developed in a previous branch